packages/clerk-js/src/ui/utils/test/createFixtures.tsx

```diff
@@ -84,22 +89,32 @@ const unboundCreateFixtures = <N extends UnpackContext<typeof ComponentContext>[
 
     const MockClerkProvider = (props: any) => {
       const { children } = props;
+
+      const componentsWithoutContext = ['UsernameSection', 'UserProfileSection'];
+      const contextWrappedChildren = !componentsWithoutContext.includes(componentName) ? (
+        <ComponentContextProvider
+          componentName={componentName}
+          props={componentContextProps}
+        >
+          {children}
+        </ComponentContextProvider>
+      ) : (
+        <>{children}</>
+      );
+
       return (
         <CoreClerkContextWrapper
           clerk={clerkMock}
           // Clear swr cache
           swrConfig={{ provider: () => new Map() }}
         >
+          <GoogleOneTapContext.Provider value={{ componentName: 'GoogleOneTap' }} />
```

Comment:
Whats the reasoning for this addition outside the other providers?

Reply:
oh whoops! Removed